docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -35,60 +35,75 @@ Required connection parameters include the following:
+  Please note that the above fields are required for the bot to be usable in Microsoft Teams. The URLs provided in the *Website*, *Privacy policy*, and *Terms of use* fields must be valid URLs.
   </Tip>
-
-  12. Fill out the *Microsoft App ID* using the client ID obtained in the previous steps, agree to the terms, and click on *Register*.
-  13. Under *Add a featured channel*, click on *Microsoft Teams*, select your Microsoft Teams solution, click on *Save* and agree to the terms when prompted.
-  14. Navigate to Microsoft Teams and then to the *Apps* tab.
-  15. Search for the *Developer Portal* app and add it to your workspace.
-  16. Open the *Developer Portal*, click on *Apps* and then on *New app*.
-  17. Fill out the *Name* for the app and click on *Add*.
-  18. Navigate to *Basic information* and fill out the required fields: *Short description*, *Long description*, *Developer or company name*, *Website*, *Privacy policy*, *Terms of use*, and *Application (client) ID*. You may also provide any additional information you wish.
-  <Tip>
-  Please note that the above fields are required to be filled out for the bot to be usable in Microsoft Teams. The URLs provided in the *Website*, *Privacy policy*, and *Terms of use* fields must be valid URLs.
-  </Tip>
-
-  19. Navigate to *App features* and select *Bot*.
-  20. Choose the *Select an existing bot* option and select the bot you created earlier in the Microsoft Bot Framework portal.
-  21. Select all of the scopes where bot is required to be used and click on *Save*.
-  22. Finally, on the navigation pane, click on *Publish* and then on *Publish to your org*.
-
+    - Navigate to *App features* and select *Bot*.
+    - Choose the *Select an existing bot* option and select the bot you created earlier in the Microsoft Bot Framework portal.
+    - Select all of the scopes where bot is required to be used and click on *Save*.
+    - Finally, on the navigation pane, click on *Publish* and then on *Publish to your org*.
+  </Step>
+</Steps>
+
+<Info>
 For the bot to be made available to the users in your organization, you will need to ask your IT administrator to approve the submission at this [link](https://admin.teams.microsoft.com/policies/manage-apps). 
 
 Once it is approved, users can find it under the *Built for your org* section in the *Apps* tab. They can either chat with the bot directly or add it to a channel. To chat with the bot in a channel, type `@<bot_name>` followed by your message.
 
 While waiting for approval, you can test the chatbot by clicking on *Preview in Teams* in the Developer Portal.
+</Info>
+
+## Usage
+
+This integration can only be used to create chatbots for Microsoft Teams via the [`CREATE CHATBOT`](/agents/chatbot) syntax. Currently, *it cannot be used as a data source for other workloads*.
+
+```sql
+CREATE CHATBOT teams_chatbot
+USING
+    database = 'teams_datasource',
+    model = 'langchain_model',
```

Comment:
Should we go with `model` here? I think the recommended way to create a chat-bot is via an agent (with skills).